web/styles/dark_theme.css

```diff
@@ -450,15 +450,15 @@
     }
 
     #search_arrows .pill,
-    .pm_recipient .pill-container .pill {
+    #compose-private-recipient.pill-container .pill {
         color: inherit;
         border: 1px solid hsl(0deg 0% 0% / 50%);
         background-color: hsl(0deg 0% 0% / 25%);
         font-weight: 600;
     }
 
     #search_arrows .pill:focus,
-    .pm_recipient .pill-container .pill:focus {
+    #compose-private-recipient.pill-container .pill:focus {
```

Comment:
I think this element name doesn't survive to the end of the PR anyway and I know you wrote this before the rename, but just wanted to mention we should try to aim to use `direct` rather than `private` when naming new things; it'll save work in the future sweep.

Reply:
Ah cool I can swap them out! It does survive to the end I'm pretty sure.